docs/MAINTAINERS.adoc

```diff
@@ -149,6 +156,140 @@ code reviewers are expected to suggest proper channels for contributors to discu
+It is perfectly reasonable for the assignee or any other reviewer to ask questions,
+but the ultimate goal of these questions should be to arrive at clear and justified action item(s);
+otherwise, the process can languish for an extended period of time.
+It is the responsibility of the assignee to steer the discussion towards concrete and justified action item(s).
+
+==== Providing confirmation that feedback has been addressed
+
+Once any requested actions have been taken, the assignee should make every reasonable effort to provide explicit confirmation that each action item has been completed.
+This gives contributors positive reinforcement and confidence that their submission is moving forward through the process,
+ultimately making them more likely to complete the process and contribute again.
+Assignees who cannot provide such confirmation in a timely fashion are strongly encouraged to remove their assignment from the pull request in order to allow another core maintainer to pick it up.
+If the assignee cannot respond in a timely fashion, the author or another core maintainer may ask the current assignee about their intentions;
+in the absence of a timely response, another core maintainer may remove the assignment.
+
+==== Negotiating scope
+
+Not every pull request reaches a state of perfection at the end of the review process.
+Sometimes, requests are made that, while justified, represent an additional amount of work that the contributor may not be willing to do.
+In some cases it is critical to complete the additional work, but in others "you aren't gonna need it" (YAGNI).
+In such cases, the assignee should make a good faith effort to negotiate with the contributor to find a reasonable middle ground that is "good enough."
+Failure to negotiate successfully can often chase contributors away.
+If the additional work is simple enough and the submission is not moving forward,
+the assignee may consider occasionally giving the contributor a lift by completing the additional work,
+though this is not expected in the general case
+and would not be fair to the assignee if a large amount of additional work is necessary.
+
+==== Voting process
+
+A pull request can often serve as a catalyst for a discussion in which several possible paths forward are identified.
+When there is no clear consensus among the core maintainers about the path forward,
+the assignee should call for a vote.
+While only core maintainers have formally binding votes, any interested parties are generally encouraged to vote, even if their votes are merely advisory.
+To avoid ambiguity, it is preferred that votes be done using https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions[Apache conventions].
+A -1 vote by a core maintainer stops a pull request in its tracks.
+This constitutes a veto, and it can neither be overruled nor overridden under normal circumstances.
```

Comment:
There was no veto before, any maintainer could have been outvoted in the developer mailing list. Personally I find vetoes, even if justified, harmful for the community. Someone can request the change and any reasonable feedback should be addressed and negotiated, but I do not believe in veto without a way to override it